pkg/minikube/bootstrapper/kubeadm/versions.go

```diff
@@ -25,6 +25,7 @@ import (
 
 	"github.com/blang/semver"
 	"github.com/golang/glog"
+	"github.com/kubernetes/minikube/pkg/version"
```

Comment:
Is this PR missing a file?

Reply:
I'm not sure.  could you please rephrase?  I am hoping this file (and the other kubeadm one) isn't from somewhere else.

Reply:
The travis test failed with:

```
Running go tests...
pkg/minikube/bootstrapper/kubeadm/versions.go:28:2: cannot find package "github.com/kubernetes/minikube/pkg/version" in any of:
	/home/travis/gopath/src/k8s.io/minikube/vendor/github.com/kubernetes/minikube/pkg/version (vendor tree)
	/home/travis/.gimme/versions/go1.12.linux.amd64/src/github.com/kubernetes/minikube/pkg/version (from $GOROOT)
	/home/travis/gopath/src/github.com/kubernetes/minikube/pkg/version (from $GOPATH)
make: *** [test] Error 1
The command "make test" exited with 2.
```

I'm not sure why this is happening though, because that package does exist as far as I know. Hmm...

Reply:
I mean.. it's not totally necessary because I feel pretty confident the version prefix isn't changing from `v` any time soon.  I just used it because I noticed it was there and figured it could (if nothing else) be clearer instead of looking like some random magic-character, you know?

Reply:
To pass the tests, this should be:

	"k8s.io/minikube/pkg/version"

Thanks!